LICENSE

```diff
@@ -1,6 +1,7 @@
 MIT License
 
 Copyright (c) 2021 Ambrose J Carr
```

Comment:
At a minimum, Ambrose should be assigning his copyright to CZI (not claiming it personally).  I believe we need to add "2022 Chan Zuckerberg Initiative Foundation" as well.

@ambrosejcarr - can you illuminate the personal copyright?  Pretty sure that is counter to your employment agreement :-)

Reply:
I think this is an artifact of github auto-creation templates, and is not intentional. 

I agree to assign my copyright to CZI or consider another entity recommended by this group.

Reply:
Could you be the one to make that change formally in the file itself? It seems like it would be unwise for me to be the one to edit that line.

Reply:
https://github.com/single-cell-data/SOMA/pull/77